src/Document.php

```diff
@@ -10,6 +10,10 @@
  */
 class Document
 {
+    const META_ATTR = '_meta';
```

Comment:
Just reverted, just wanted to avoid having up to 6 refs to the same string in the source file. Although not using consts will be more consistant with the coding style of the repo.